src/js/common/components/Widgets/ReadMore.jsx

```diff
@@ -27,6 +30,43 @@ export default class ReadMore extends Component {
   toggleLines (event) {
     event.preventDefault();
     const { readMore } = this.state;
+    const { location: { pathname: currentPathname } } = window;
+    const currentPage = lookupPageNameAndPageTypeDict(currentPathname);
+
+    const {
+      eventName = 'clickShowMoreAboutPolitician',
+      politicianIdShowMore,
+      politicianNameShowMore,
+    } = this.props;
+
+    console.log("Right here!",readMore, politicianNameShowMore, politicianIdShowMore);
+
+    TagManager.dataLayer({
+    dataLayer: {
+      event: 'click',
+      actionDetails: {
+        actionType: 'showMore',
+        buttonId: eventName
+        },
+      pageDetails: {
+        pageName: currentPage.pageName,
+        pageType: currentPage.pageType,
+        pathname: currentPathname,
+      },
+      politicianDetails: {
+        politicianWeVoteId: politicianIdShowMore,
```

Comment:
This becomes:
```
politicianWeVoteId,
politicianName: PoliticianStore.getPoliticianName(politicianWeVoteId),
```
All the variables we want to track are here:
https://docs.google.com/spreadsheets/d/1KO4_AOmddKvIpA79XrZdaVBdRkcP1HitqrvQkCo2A34/edit?pli=1&gid=1828322090#gid=1828322090
For the other politician variables, I would pull the entire politician with PoliticianStore.getPoliticianByWeVoteId prior to the dataLayer, and then reference the values in that politician dict.